### PR TITLE
CI/Build: Pre-Release Changes for 24.08

### PR DESCRIPTION
**Pull request by pvijayakrish**
#### Checklist
- [x] I have read the [Contribution guidelines](#../../CONTRIBUTING.md) and signed the [Contributor License
Agreement](https://github.com/NVIDIA/triton-inference-server/blob/master/Triton-CCLA-v1.pdf)
- [x] PR title reflects the change and is of format `<commit_type>: <Title>`
- [x] Changes are described in the pull request.
- [ ] Related issues are referenced.
- [ ] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [ ] Added [test plan](#test-plan) and verified test passes.
- [ ] Verified that the PR passes existing CI.
- [ ] I ran pre-commit locally (`pre-commit install, pre-commit run --all`)
- [ ] Verified copyright is correct on all changed files.
- [ ] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [ ] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [x] build
- [x] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

### CODE REVIEW

**Review by nvda-mesharma — DISMISSED** [review]

File: build.py [hunk 1]
```diff
@@ -69,10 +69,10 @@
...
 #
 TRITON_VERSION_MAP = {
-    "2.49.0dev": (
-        "24.08dev",  # triton container
-        "24.07",  # upstream container
-        "1.18.1",  # ORT
+    "2.49.0": (
+        "24.08",  # triton container
+        "24.08",  # upstream container
+        "1.19.0",  # ORT
```
Comment (mc-nv) [inline build.py:75]: This version is not confirmed yet.

File: README.md [hunk 2]
```diff
@@ -41,236 +41,8 @@ ___
...
-For questions, we recommend posting in our community
-[GitHub Discussions.](https://github.com/triton-inference-server/server/discussions)
-
-## For more information
-
-Please refer to the [NVIDIA Developer Triton page](https://developer.nvidia.com/nvidia-triton-inference-server)
-for more information.
+> [!Warning]
+>
+> ##### LATEST RELEASE
```
Comment (mc-nv) [inline README.md:46]: Isn't this line redundant for "warning" message?

File: build.py (hunk 1 above)
Comment (mc-nv) [reply build.py:75]: Ok so this version is OK for Linux. 
For windows we will have to carry a side branch with version `1.18.1`

**Comment by oandreeva-nv** [comment]
Please update PR description according to a provided template

File: Dockerfile.win10.min [hunk 3]
```diff
@@ -37,7 +37,7 @@ RUN choco install unzip -y
 #
 # Installing TensorRT
 #
-ARG TENSORRT_VERSION=10.2.0.19
+ARG TENSORRT_VERSION=10.3.0.26
```
[the remark this answers is not in the document]
Comment (pvijayakrish) [reply Dockerfile.win10.min:40]: Agreed!
I believe also [CUDNN version](https://github.com/triton-inference-server/server/blob/368d12f6209e0dd2c2571d2606671b24fce38603/Dockerfile.win10.min#L54C2-L54C27) should be different.

**Review by nv-kmcgill53 — APPROVED** [review]

**Review by nv-kmcgill53 — APPROVED** [review]

**Pull request merged**